Upgrade to using uv instead of pip (#111)
PECOS-packages/PECOS: DEVELOPMENT.md

Reviewer: Tried the instructions, they work. Now that we are using `uv`, mentions of venv and virtual environment both here and in Makefile (make venv) feel unnecessary. I will let you take an executive decision.

   ```sh
   make venv
   ```

>    or manually set up a Python [virtual environment using uv](https://docs.astral.sh/uv/getting-started/installation/) for develop of this project's code.

Reviewer: ```suggestion
   or manually set up a Python [virtual environment using uv](https://docs.astral.sh/uv/getting-started/installation/) for development.
```

   To do so, navigate to the root of this project and run:

   ```sh

>    python -m pip install --upgrade uv

Reviewer: `uv` is best installed systemwide, I'd point people to https://docs.astral.sh/uv/getting-started/installation/ instead of suggesting `pip install` here.
